google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -139,11 +140,10 @@ def create_finding(self, finding, organization_id=None, source_id=None,
             # beta api
             try:
                 LOGGER.debug('Creating finding with beta api.')
-                response = self.repository.findings.create(
+                response = self.repository.findings.patch(
```

Comment:
In line 102 above, you need to pass an additional parameter to init: key_field='name'

Then update the call to patch like this:

response = self.repository.findings.patch(
    '{}/findings/{}'.format(source_id, finding_id),
    findings)

Reply:
thanks for the quick review and feedback!